sparta/test/ValidValue/ValidValue_test.cpp

```diff
@@ -83,6 +83,14 @@ int main()
         item.emplace_back(i, "test");
     }
 
+    std::vector<sparta::utils::ValidValue<Foo>> item2;
+    for(uint32_t i = 0; i < 100; ++i) {
+
```

Comment:
A couple of spurious changes got in here...

Reply:
Fixed